examples/nested_decision_execution/workflow.py

```diff
@@ -0,0 +1,232 @@
+            "kwargs": {
+                "time_per_step": "day",
+                "inner_executor": {
+                    "class": "NestedExecutor",
+                    "module_path": "qlib.backtest.executor",
+                    "kwargs": {
+                        "time_per_step": "30min",
+                        "inner_executor": {
+                            "class": "SimulatorExecutor",
+                            "module_path": "qlib.backtest.executor",
+                            "kwargs": {
+                                "time_per_step": "5min",
+                                "generate_report": True,
+                                "verbose": True,
+                                "indicator_config": {
+                                    "show_indicator": True,
+                                },
+                            },
+                        },
+                        "inner_strategy": {
+                            "class": "TWAPStrategy",
+                            "module_path": "qlib.contrib.strategy.rule_strategy",
+                        },
+                        "generate_report": True,
+                        "indicator_config": {
+                            "show_indicator": True,
+                        },
+                    },
+                },
+                "inner_strategy": {
+                    "class": "SBBStrategyEMA",
+                    "module_path": "qlib.contrib.strategy.rule_strategy",
+                    "kwargs": {
+                        "instruments": market,
+                        "freq": "1min",
+                    },
+                },
+                "track_data": True,
+                "generate_report": True,
+                "indicator_config": {
+                    "show_indicator": True,
+                },
+            },
+        },
+        "backtest": {
+            "start_time": "2020-01-01",
+            "end_time": "2020-12-31",
+            "account": 100000000,
+            "exchange_kwargs": {
+                "freq": "1min",
+                "limit_threshold": 0.095,
+                "deal_price": "close",
+                "open_cost": 0.0005,
+                "close_cost": 0.0015,
+                "min_cost": 5,
+            },
+        },
+    }
+
+    def _init_qlib(self):
+        """initialize qlib"""
+        # provider_uri_day = "/data/stock_data/huaxia/qlib"
+        # provider_uri_1min = "/data2/stock_data/huaxia_1min_qlib"
+        provider_uri_day = "~/.qlib/qlib_data/cn_data"  # target_dir
+        GetData().qlib_data(target_dir=provider_uri_day, region=REG_CN, version="v2", exists_skip=True)
+        provider_uri_1min = HIGH_FREQ_CONFIG.get("provider_uri")
+        GetData().qlib_data(
+            target_dir=provider_uri_1min, interval="1min", region=REG_CN, version="v2", exists_skip=True
+        )
+        provider_uri_map = {"1min": provider_uri_1min, "day": provider_uri_day}
```

Comment:
The current design is quite strange.
`provider_uri` is useless now.
Why can't we merge the feature of `provider_uri_map`  into  `provider_uri`

Reply:
Merge Pengrong's PR #580